src/notification-hub/azext_notification_hub/_params.py

```diff
@@ -81,14 +81,14 @@ def load_arguments(self, _):
         c.argument('notification_hub_name', options_list=['--name', '-n'], help='The notification hub name.')
         c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
         c.argument('tags', tags_type)
-        c.argument('sku_name', arg_type=get_enum_type(['Free', 'Basic', 'Standard']), help='Name of the notification hub sku')
+        c.ignore('sku_name')
```

Comment:
This would be a breaking change and user would meet errors. Just want to confirm. If the usage is quite low, it's fine to do it.